src/components/reusables/Form.tsx

```diff
@@ -19,6 +19,8 @@ export interface FormProps {
 }
 
 export default class Form extends React.Component<FormProps, {}> {
+  private messageRef = React.createRef<HTMLParagraphElement>();
```

Comment:
The new way to declare refs. It is passed down to the `p` in line 44 and then referenced as `this.messageRef.current`.